azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/service/sdk/CallingSDKWrapper.kt

```diff
@@ -262,6 +262,8 @@ internal class CallingSDKWrapper(
         return call.mute(context)
     }
 
+    private fun callIsNotSetup() = callClient == null && deviceManagerCompletableFuture == null
+
```

Comment:
private functions can be moved to end of file?

public functions first
private last